lib/r10k/module/local.rb

```diff
@@ -11,10 +11,14 @@ def self.implement?(name, args)
     args.is_a?(Hash) && (args[:local] || args[:type].to_s == 'local')
   end
 
-  def version
+  def self.statically_defined_version(name, args)
     "0.0.0"
   end
 
+  def version
+    self.statically_defined_version(self.name, nil)
```

Comment:
shouldn't this be `self.class.statically_defined_version(self.name, nil)`

Reply:
maybe/probably.. I've never been solid when to use `self` vs. not, etc. 😅